.circleci/config.yml

```
@@ -24,4 +79,9 @@ workflows:
       - maven/test:
           executor: jdk
       - build:
-          context: SonarCloud
\ No newline at end of file
+          context: SonarCloud
+      - nativeImage
+      - deploy-k8s:
+          requires:
+            - maven/test
+            - build
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)